recipes/r-changepoint/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: r-changepoint
+  version: "2.2.1"
+
+source:
+  fn: changepoint_2.2.1.tar.gz
+  url:
+    - http://cran.r-project.org/src/contrib/changepoint_2.2.1.tar.gz
+    - http://cran.r-project.org/src/contrib/Archive/changepoint/changepoint_2.2.1.tar.gz
+
+build:
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r
+    - r-zoo >=0.9_1
+    - gcc # [not win]
```

Comment:
For osx we recommend to include `- llvm # [osx]`. libgcc is not needed in this case.